Hymns/Favorites/FavoritedEntity.swift

```diff
@@ -5,30 +5,33 @@ class FavoriteEntity: Object, Identifiable {
     @objc dynamic var primaryKey: String!
```

Comment:
Like I said above, you should create a TagEntity, and have FavoriteEntity subclass it with a special hardcoded string as the tag. So both entities should exist, and we can still use FavoriteEntity as we use it today

Reply:
here, it's not about abstraction. It's about being DRY. You don't want to have to repeat your hardcoded string everywhere as you do now, because that is brittle and easy to break. I basically implemented what I was saying here: https://github.com/HymnalDreamTeam/Hymns-iOS/pull/239. Let me know if that makes sense to you.